src/components/StorageOverview/DataResiliency/DataResiliency.js

```diff
@@ -0,0 +1,100 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Icon, UtilizationBar } from 'patternfly-react';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { getCapacityStats } from '../../../selectors/prometheus/selectors';
+
+const DataResiliencyStatusBody = ({ status }) =>
+  status ? (
+    <Fragment>
+      <div className="kubevirt-data-resiliency__status-title-ok">Your Data Is Resilient</div>
+      <div className="kubevirt-data-resiliency__icon-ok">
+        <Icon type="fa" name="check-circle" size="2x" />
+      </div>
+    </Fragment>
+  ) : (
+    <Fragment>
+      <div className="kubevirt-data-resiliency__icon-error">
+        <Icon type="fa" name="exclamation-triangle" size="2x" />
+      </div>
+      <div className="kubevirt-data-resiliency__status-title-error">Some Error Occurred</div>
```

Comment:
can we pass the error message here?

also  just `Error Occured` or just `Error` sounds better IMO

Reply:
what error message @suomiy

Reply:
that is what the user would say :) when the UI says `Some Error Occurred`

Reply:
what should I write ..can you tell ?

Reply:
Prometheus just returns integer value which indicates state of storage. So it seems like there is no way to get some meaningful error message right now. This is not blocking IMO and we can merge once the PR is rebased

Reply:
okay